### PR TITLE
code-generator: add ut in ./staging/src/k8s.io/code-generator/generators

### PR DESCRIPTION
**Pull request by tukwila**
What type of PR is this?

/kind cleanup

What this PR does / why we need it:

[UT]Increase ut for code-generator
https://github.com/kubernetes/kubernetes/issues/119289
k8s.io/code-generator/cmd/conversion-gen/generators coverage: 0.5% of statements

Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?
```
NONE
````
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```
NONE
```

### CODE REVIEW

**Comment by tukwila** [comment]
@sttts Could please review this UT again? thank you.

File: staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go [hunk 1]
```diff
@@ -0,0 +1,328 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (sttts) [inline staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go:2]: 2023

Comment (tukwila) [reply staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go]: Corrected.

File: staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+	}
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			r := extractTag(tc.comments)
+			assert.Equal(t, tc.expect, r)
+		})
+	}
+}
+
+func Test_extractExplicitFromTag(t *testing.T) {
```
Comment (sttts) [inline staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go:98]: not sure testing all these functions has so much value. They are just one-liner variants of `types.ExtractCommentTags`. The later is unit tested in gengo. Here, we only project to one tag name.

Looking further down in conversion.go, what @aojea meant with coverage in https://github.com/kubernetes/kubernetes/issues/119289 is about the business logic of the generates. For instance `getManualConversionFunctions` is pretty complex. But also the whole package could be unit tested. The functions here in the PR are rather just one or two-liner helpers which are obvious what they do.

**Pull request closed without merging**